hr_expense_cancel/i18n/hr_expense_cancel.pot

```diff
@@ -0,0 +1,27 @@
+# Translation of Odoo Server.
```

Comment:
This file is not needed anymore and it's convenient to remove to avoid outdated files.